[2022-04-19 23:59:12,313] {processor.py:163} INFO - Started process (PID=1579) to work on /opt/airflow/dags/gcs_to_bq_dag.py
[2022-04-19 23:59:12,313] {processor.py:642} INFO - Processing file /opt/airflow/dags/gcs_to_bq_dag.py for tasks to queue
[2022-04-19 23:59:12,314] {logging_mixin.py:109} INFO - [2022-04-19 23:59:12,314] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/gcs_to_bq_dag.py
[2022-04-19 23:59:12,715] {logging_mixin.py:109} INFO - [2022-04-19 23:59:12,713] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/gcs_to_bq_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/gcs_to_bq_dag.py", line 42, in <module>
    "sourceUris": [f"gs://{BUCKET}/raw/{parquet_file}"],
NameError: name 'parquet_file' is not defined
[2022-04-19 23:59:12,715] {processor.py:654} WARNING - No viable dags retrieved from /opt/airflow/dags/gcs_to_bq_dag.py
[2022-04-19 23:59:12,726] {processor.py:171} INFO - Processing /opt/airflow/dags/gcs_to_bq_dag.py took 0.414 seconds
[2022-04-19 23:59:42,870] {processor.py:163} INFO - Started process (PID=1638) to work on /opt/airflow/dags/gcs_to_bq_dag.py
[2022-04-19 23:59:42,871] {processor.py:642} INFO - Processing file /opt/airflow/dags/gcs_to_bq_dag.py for tasks to queue
[2022-04-19 23:59:42,871] {logging_mixin.py:109} INFO - [2022-04-19 23:59:42,871] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/gcs_to_bq_dag.py
[2022-04-19 23:59:43,304] {logging_mixin.py:109} INFO - [2022-04-19 23:59:43,302] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/gcs_to_bq_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/gcs_to_bq_dag.py", line 42, in <module>
    "sourceUris": [f"gs://{BUCKET}/raw/{parquet_file}"],
NameError: name 'parquet_file' is not defined
[2022-04-19 23:59:43,305] {processor.py:654} WARNING - No viable dags retrieved from /opt/airflow/dags/gcs_to_bq_dag.py
[2022-04-19 23:59:43,315] {processor.py:171} INFO - Processing /opt/airflow/dags/gcs_to_bq_dag.py took 0.446 seconds
